lib/classes/Swift/Mime/Headers/PathHeader.php

```diff
@@ -32,16 +32,25 @@ class Swift_Mime_Headers_PathHeader extends Swift_Mime_Headers_AbstractHeader
      */
     private $emailValidator;
 
+    /**
+     * The address encoder.
+     *
+     * @var Swift_AddressEncoder
+     */
```

Comment:
phpdoc can be removed

Reply:
@fabpot but the type should stay. phpstorm use this for autocompletion :)

Reply:
I don't use phpstorm, but IIRC, I think that the type hint in the constructor and the assignment is enough. No need to repeat it here.